lib/data/network/dio_cache_option.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:dio_cache_interceptor/dio_cache_interceptor.dart';
+import 'package:dio_cache_interceptor_hive_store/dio_cache_interceptor_hive_store.dart';
+import 'package:flutter/foundation.dart';
+import 'package:matrix/matrix.dart';
+import 'package:path_provider/path_provider.dart';
+
+class DioCacheOption {
+  static const String _hiveBoxName = "twake_dio_cache_hive_store";
+  static const Duration _maxStale = Duration(days: 7);
```

Comment:
the duration 7 should be in app config imo

Reply:
Yes I change to 3 days